cassandra/tablets.py

```diff
@@ -0,0 +1,46 @@
+# Experimental, this interface and use may change
+import bisect
+from threading import Lock
+
+
+class Tablet(object):
+    """
+    Represents a single ScyllaDB tablet.
+    """
+    table_id = None
+    last_token = 0
+    tablet_count = 0
```

Comment:
It is the question if we want to have here all the info that is in system.tablets, I am not sure about that

Reply:
We don't need to hold all the info, we just needs representation that fullfills driver's needs (sending queries, logging)

Reply:
But for now we also do not use info about `stage` and `new_replicas`, do you think I should remove this fields too?

Reply:
Afaik those will be required later, won't they? If so, you can leave them or add a comment about it.
That's why I asked if we'll need `table_id` in the future. If not, it can be removed.

Reply:
I am not sure about that to be honest

Reply:
Ok, then let's remove `table_id`, `tablet_count` and leave `stage` and `new_replicas`. If it turns out that we do need `table_id` we can add it back.

Reply:
`table_id` and `tablet_count` removed